SofaKernel/modules/SofaBaseTopology/src/SofaBaseTopology/EdgeSetGeometryAlgorithms.inl

```diff
@@ -899,5 +899,68 @@ void EdgeSetGeometryAlgorithms<DataTypes>::initPointAdded(PointID index, const c
     }
 }
 
+template<class DataTypes>
+bool EdgeSetGeometryAlgorithms<DataTypes>::computeEdgeSegmentIntersection(EdgeID edgeID,
+    const sofa::defaulttype::Vec<3, double>& a,
+    const sofa::defaulttype::Vec<3, double>& b,
+    double &baryCoef)
+    //intersection=edge[0]+baryCoef*(edge[1]-edge[0])
+{
+    double EPS = 1e-05;
```

Comment:
const or even constexpr

(I dont speak of the value of epsilon itself as it is the same value elsewhere in this file, but 1e-05 seems almost too big IMO)